stetho/src/main/java/com/facebook/stetho/inspector/elements/android/TextViewDescriptor.java

```diff
@@ -5,21 +5,30 @@
 import android.text.Editable;
 import android.text.TextWatcher;
 import android.widget.TextView;
-
 import com.facebook.stetho.common.Util;
 import com.facebook.stetho.inspector.elements.AttributeAccumulator;
 import com.facebook.stetho.inspector.elements.ChainedDescriptor;
 
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.IdentityHashMap;
 import java.util.Map;
 
 final class TextViewDescriptor extends ChainedDescriptor<TextView> {
   private static final String TEXT_ATTRIBUTE_NAME = "text";
 
+  private final MethodInvoker mMethodInvoker;
```

Comment:
this MethodInvoker class is pretty cool, but we don't (yet?) want to support arbitrary set___() calls. We only emit a very small subset of attributes to the element tree like 'id' and 'text', and so we should only support writing to that small subset. For now it should be sufficient to just check that name='text' and then call the setText() method. Supporting all get___/set___ method pairs, along with the marshaling of types to and from strings, would be a giant can of worms.